### DISCUSSION TITLE
How to remove unused wallets from the list?

### DISCUSSION THREAD

**Discussion by web3456** [WalletConnect/walletconnect-monorepo#4042] [Developer Support] [answered]
How to remove unused wallets from the list?

I can select only recommended here
``const web3Modal = new Web3Modal({
  projectId: DEFAULT_PROJECT_ID,
  themeMode: "light",
  walletConnectVersion: 2,
  explorerRecommendedWalletIds: [
    'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96',
    '4622a2b2d6af1c9844944291e5e7351a6aa24cd7b23099efac1b2fd875da31a0',
    'ecc4036f814562b41a5268adc86270fba1365471402006302e70169465b7ac18',
  ],
});``
is any opportunity to remove unused wallets?
<img width="399" alt="Screenshot 2023-12-13 at 20 30 56" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/153770518/1374fc79-d430-4857-9737-bf6d109f73c2">

**Comment by glitch-txs** [comment, marked as answer]
Hi, please update to Web3Modal v3 since v2 has been deprecated, you can select the wallets you want to show in this version: 
https://docs.walletconnect.com/web3modal/about
Thanks!
